tuxemon/core/npc.py

```diff
@@ -244,11 +244,14 @@ def load_sprites(self):
         # at the same time, so that way they'll always be in sync with each other.
         self.moveConductor.add(self.sprite)
 
-    def get_sprites(self):
+    def get_sprites(self, layer):
```

Comment:
i think that the layer attribute should be on the NPC.  its not common, but occasionally it is useful to change NPC layers to add depth to the world.  In that case, the layer attribute would be unique to each sprite/NPC/entitiy.  Please consider moving the layer attribute to the NPC class.

Reply:
I'm not sure how that will work with maps specifying the layer to draw sprites, even if the npc overrides it I think it'll end up kinda ugly.

Reply:
yeah.  its not worth thinking about too much now, but i think we should have a comment so that we don't keep adding to this kludge.  its a fine kludge, but when/if NPCs can move to different layers, i want this refactored and removed.